openslides_backend/action/actions/user/set_password_self.py

```diff
@@ -42,14 +39,10 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
     def check_permissions(self, instance: Dict[str, Any]) -> None:
         if self.auth.is_anonymous(self.user_id):
             raise ActionException("Can't set password for anonymous")
-
         instance["id"] = self.user_id
-        if is_temporary(self.datastore, instance):
-            if has_perm(
-                self.datastore,
-                self.user_id,
-                Permissions.User.CAN_CHANGE_OWN_PASSWORD,
-                instance["meeting_id"],
-            ):
-                return
-            raise MissingPermission(Permissions.User.CAN_CHANGE_OWN_PASSWORD)
+        user = self.datastore.get(
+            FullQualifiedId(self.model.collection, self.user_id),
+            ["can_change_own_password"],
+        )
+        if not user.get("can_change_own_password"):
+            raise PermissionDenied("Missing Permission: can_change_own_password")
```

Comment:
```suggestion
            raise PermissionDenied("You cannot change your password")
```
since it is not really a permission anymore and the other message might lead users to searching for the permission which no longer exists.

Reply:
Done.